Assets/Resources/Scripts/PlayerMovementScript.js

```diff
@@ -0,0 +1,46 @@
+﻿// Player movement (of sorts), attached to player game object
+
+#pragma strict
+
+// speed of player character, can override in editor
+var speed : float = 3;
+
+// Rigidbody2D component of player game object
+private var rigidBody : Rigidbody2D;
+
+// Called at start, grabs the rigidbody component for later use
+// (so there is no need to point the script at it in the editor).
+function Start() {
+
+	this.rigidBody = gameObject.GetComponent.<Rigidbody2D>();
```

Comment:
gameObject ei ole määritelty tässä

`MissingComponentException: There is no 'Rigidbody2D' attached to the "GameObject" game object, but a script is trying to access it.
You probably need to add a Rigidbody2D to the game object "GameObject". Or your script needs to check if the component is attached before using it.
UnityEngine.Rigidbody2D.set_velocity (Vector2 value) (at C:/buildslave/unity/build/artifacts/generated/common/modules/Physics2D/Physics2DBindings.gen.cs:1384)
PlayerMovementScript.FixedUpdate () (at Assets/Resources/Scripts/PlayerMovementScript.js:44)`